src/cc_toolchain/cc_toolchain_config.bzl

```diff
@@ -184,14 +184,14 @@ def _make_common_features(ctx):
         enabled = True,
     )
 
-    result["warnings_feature"] = feature(
-        name = "warnings",
+    result["warnings_as_errors_feature"] = feature(
```

Comment:
Just to make sure I'm understanding this, we still need the `warnings_as_errors_feature` to actually turn warnings into errors, and everything in the `bazelrc` is just determining what warnings we have enabled?

Reply:
Yes, I can't have -Werror in the bazelrc, because it can't be overridden by the `copts=["-Wnoerror"]` in the legacy and 3rd party code, it has to live in the toolchain unfortunately.

Reply:
I think that's fine, just wanted to make sure I understood.

Reply:
Can you maybe add that as a comment somewhere just to be super clear to future build-systems people?

Reply:
added comment